Dockerfile

```diff
@@ -36,13 +34,16 @@ RUN yarn && yarn tsc && npm prune -production --legacy-peer-deps
 RUN rm -rf ./node_modules/typescript
 
 
-
-FROM nikolaik/python-nodejs:python3.12-nodejs20-alpine as final
+FROM nikolaik/python-nodejs:python3.13-nodejs22-slim as final
 
 RUN corepack enable
 
 # Use tini for NodeJS application https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#handling-kernel-signals
-RUN apk add --no-cache tini curl libmagic
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    tini \
+    curl \
+    libmagic-dev \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
why?

Reply:
This are dependencies that are needed to either run, setup or manage processes in the docker image.
 I am not sure what you meant by why. what is the specific concern you would like for me to address